modules/troubleshooting-disabling-autoreboot-mco.adoc

```diff
@@ -23,7 +23,7 @@ Pausing a machine config pool stops all system reboot processes and all configur
 .Procedure
 . To pause the autoreboot process after machine config changes are applied:
 
-* As root, update the `spec.paused` field to `true` in the MachineConfigPool CustomResourceDefinition (CRD).
+* As root, update the `spec.paused` field to `true` in the `MachineConfigPool` custom resource definition (CRD).
```

Comment:
No one outside of Operator developers will ever modify a CRD. This is a CR object instead.